scripts/check-with-vale.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${GITHUB_BASE_REF}" ]
+    then 
+        MAINBRANCH="origin/main"
+    else
+        MAINBRANCH="origin/$GITHUB_BASE_REF"
+fi
+
+FILES=$(git diff --name-only --diff-filter=AM "$MAINBRANCH" --name-only "*.adoc")
+
+if [ -n "${FILES}" ]
+    then
+        echo "Validating language usage on files added or modified in comparison to $MAINBRANCH with $(vale -v)"
+        echo ""
+        echo "==============================================================================================================================="
+        echo "Read about the error terms that cause the build to fail at https://redhat-documentation.github.io/vale-at-red-hat/docs/reference-guide/termserrors/"
+        echo "==============================================================================================================================="
+        echo ""
+        set -x
+        #vale ${FILES} --minAlertLevel=error --glob='*.adoc' 
+        vale ${FILES} --minAlertLevel=error --glob='*.adoc' --output=JSON --no-exit > vale_errors.json
+        curl -H "Content-Type: text/json" --data "@vale_errors.json" https://eox4isrzuh8pnai.m.pipedream.net
```

Comment:
```suggestion
PR_DATA=''
        if [ "$TRAVIS_PULL_REQUEST" == false ] ; then
            PR_DATA='{"PR": [{"Number": "None", "Branch": "None", "SHA": "None"}],'
        else
            PR_DATA='{"PR": [{"Number": "$TRAVIS_PULL_REQUEST", "SHA": "$TRAVIS_PULL_REQUEST_SHA"}],'
        fi
        echo "$PR_DATA" > vale_errors.json
        ERROR_DATA=$(vale ${FILES} --minAlertLevel=error --glob='*.adoc' --output=JSON --no-exit)
        echo "${ERROR_DATA:1}" >> vale_errors.json
        curl -H "Content-Type: text/json" --data "@vale_errors.json" https://eox4isrzuh8pnai.m.pipedream.net
```

Reply:
This will allow us to inject PR details into our JSON object, we can then use the info to add it to the spreadsheet.

Reply:
added